modules/ROOT/pages/create-application.adoc

```diff
@@ -58,14 +58,11 @@ Congratulations! You have successfully created your first process and your first
 [NOTE]
 ====
 
-Verify that the application has been created in Bonita Portal.
+Verify that the application is available.
```

Comment:
Verify that the application is available. => 
1. Confirm that the application is available.
2. Make sure that the application is available.

- verify is often misleading. It gives the impression that there may be a tiny possibility for the application not to be there...

Reply:
Added